### DISCUSSION TITLE
How to change bind port

### DISCUSSION THREAD

**Discussion by yonas** [ergo-services/ergo#184] [Q&A]
How do I start two nodes on the same server?

**Comment by halturin** [comment]
You can start as many nodes as you want. Node chooses free port automatically 

But you may specify it using "Acceptors" option

**Comment by yonas** [comment]
@halturin I get this error on the second run:

```sh
❯ go run ./cmd 
panic: unable to register node: resource is taken

goroutine 1 [running]:
main.main()
        /git/mynode/cmd/mynode.go:43 +0x334
exit status 2
```

**Reply by halturin** [reply]
seems you use the same node name. it must be unique within a host.
please, read the documentation https://docs.ergo.services/basics/node#starting-a-node

> The node's name must be unique on the host. This means that two nodes with the same name cannot be running on the same host.

**Reply by yonas** [reply]
Thanks, got a little further:

```sh
❯ go run ./cmd -name example@localhost
1726233645488331714 [info] C7881647: application 'system_app' (permanent) started
1726233645488644554 [error] C7881647: unable to start application 'observer_app':listen tcp 127.0.0.1:9911: bind: address already in use
panic: listen tcp 127.0.0.1:9911: bind: address already in use

goroutine 1 [running]:
main.main()
        /git/mynode/cmd/mynode.go:43 +0x334
exit status 2
```

**Reply by halturin** [reply]
in that example the observer app is included and uses port 9911. disable this app or specify another port number in the observer options.
